maquetado/scss/index.scss

```diff
@@ -41,3 +41,23 @@
   font-size: 14px;
   line-height: 17px;
 }
+
+.navbar {
+  align-items: center;
+  background-color: $white;
+  border-top: solid 6px $cerulean;
+  box-shadow: 0 4px 10px 0 $opaque-black;
+  display: flex;
+  justify-content: space-between;
+  left: 0;
+  max-width: 100%;
```

Comment:
Use max-width when you have a absolute limit for element
change for width: 100%